tests/unit/s2n_client_hello_test.c

```diff
@@ -1250,7 +1281,7 @@ int main(int argc, char **argv)
             .suites = client_cipher_suites,
         };
 
-        const struct s2n_signature_scheme* const client_sig_scheme_pref_list[] = {
+        const struct s2n_signature_scheme *const client_sig_scheme_pref_list[] = {
```

Comment:
It seems weird to put the * on const and not on s2n_signature_scheme, but maybe this is fine/correct?

Reply:
Yeah this is correct, or at least consistent. 

```c
const struct s2n_signature_scheme *const client_sig_scheme_pref_list[]
^^^^                                                      ^^^^
   |                                                               | --- This const means the pointers themselves can't change
   | --- This const means that the struct s2n_signature_scheme are not modifiable 
```

The second `const` must come after the `*`. If we removed it it would be clear that the formatting is correct:

```c
const struct s2n_signature_scheme *client_sig_scheme_pref_list[]
```

https://matt.sh/sytycc